docker/from-source/Debian

```diff
@@ -2,19 +2,23 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-FROM debian:buster
+FROM debian:stable
 
 # get the latest fixes
 # install LibreOffice run-time dependencies
 # install adduser, findutils, openssl and cpio that we need later
 # install an editor
 # tdf#117557 - Add CJK Fonts to Collabora Online Docker Image
+RUN apt-get clean && apt-get update
```

Comment:
why do you run apt-get update twice?

Reply:
And I wonder why `apt-get clean` is important here.

Reply:
@thebearon @timar  I was getting this error while building the image, so I had to put in the line 12
![remote](https://user-images.githubusercontent.com/31562363/196706942-0f93b8eb-3ff1-497a-bfd4-effd98098481.jpg)

Reply:
Okay I fixed it now